assets/js/googlesitekit/datastore/site/urls.js

```diff
@@ -24,6 +24,12 @@ import { CORE_SITE } from './constants';
 import { getLocale } from '../../../util';
 const { createRegistrySelector } = Data;
 
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+import isNumber from 'lodash/isNumber';
```

Comment:
I'm… surprised this isn't flagged by WordPress's ESLint, but we put imports in this order:

1. External
2. WordPress
3. Internal

Can you move this one up? Sorry ESLint didn't catch it! 😓